packages/app/src/app/overmind/namespaces/dashboard/state.ts

```diff
@@ -32,13 +32,12 @@ export const state: State = {
     blacklistedTemplates: [],
     search: '',
   },
-  isTemplateSelected: state => templateName => {
-    return state.filters.blacklistedTemplates.indexOf(templateName) === -1;
-  },
-  getFilteredSandboxes: state => sandboxes => {
-    const orderField = state.orderBy.field;
-    const orderOrder = state.orderBy.order;
-    const blacklistedTemplates = state.filters.blacklistedTemplates;
+  isTemplateSelected: ({ filters }) => templateName =>
+    filters.blacklistedTemplates.indexOf(templateName) === -1,
```

Comment:
Use `Array.includes` instead

```suggestion
    !filters.blacklistedTemplates.includes(templateName),
```